### PR TITLE
Doc: Add Chinese doc  of simulat-time-chaos-on-physical-nodes

### PR DESCRIPTION
**Pull request by Andrewmatilde**
Signed-off-by: Andrewmatilde <davis6813585853062@outlook.com>

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+```bash
+chaosd attack clock -p $PID -t 11s
+```
+
+### 模拟时间故障的相关配置
+
+| 配置项 | 类型 | 说明 | 默认值 | 必要项 | 例子 |
+| --- | --- | --- | --- | --- | --- |
+| timeOffset | string | 指时间的偏移量。 | None | 是 | `-5m` |
```
Comment (WangXiangUSTC) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md:74]:
```suggestion
| timeOffset | string | 指定时间的偏移量。 | None | 是 | `-5m` |
```

**Review by WangXiangUSTC — APPROVED** [review]
rest LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md (hunk 1 above)
Comment (Andrewmatilde) [reply i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md]: Fixed.

**Comment by WangXiangUSTC** [comment]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**